src/binaries/trezord-go/bin/download.sh

```diff
@@ -3,5 +3,10 @@ set -e
 
 # TODO: fetch trezord-go from nix
 
+SYSTEM_ARCH=$(uname -m)
 # Bridge needs an older glibc so we are pinning to nixos-21.05 (stable) as of 2021-07-02
-nix-shell -p autoPatchelfHook -I "nixpkgs=https://github.com/NixOS/nixpkgs/archive/e9148dc1c30e02aae80cc52f68ceb37b772066f3.tar.gz" --run "autoPatchelf trezord-*"
+if [[ $SYSTEM_ARCH == x86_64* ]]; then
```

Comment:
It is a question if this is necessary, as the `"autoPatchelf trezord-*"` could catch everything, even `trezord-*-arm64`. In ARM images, there should also not be any "non-ARM" bridges (I guess?)